[Docs] Add risk mitigation links (#870)
mckinsey/vizro: vizro-ai/docs/pages/user-guides/customize-vizro-ai.md

    | API key           | `OPENAI_API_KEY`                      |
    | Base API URL      | `OPENAI_API_BASE`                     |

>     To use OpenAI with Vizro-AI, you must have an account with paid-for credits available. None of the free accounts will suffice. You can check [all available OpenAI models including pricing on their website](https://platform.openai.com/docs/models). This will also explain which version the below string acronyms currently point to. Before usage, please review OpenAI's guidelines on risk mitigation to understand potential model limitations and best practices.

Reviewer: ```suggestion
    To use OpenAI with Vizro-AI, you must have an account with paid-for credits available. None of the free accounts will suffice. [Check the OpenAI models and pricing on their website](https://platform.openai.com/docs/models). Before using a model, please review OpenAI's guidelines on risk mitigation to understand potential model limitations and best practices.
```

>     More details on responsible usage can be found on this [link](https://platform.openai.com/docs/models).

Reviewer: ```suggestion
    [See the OpenAI site for more details on responsible usage](https://platform.openai.com/docs/models).
```

Reviewer 2: Should we not link to this: https://platform.openai.com/docs/guides/safety-best-practices. I have only browsed this, but it seems more relevant?

    | API key           | `ANTHROPIC_API_KEY`                       |
    | Base API URL      | `ANTHROPIC_API_URL`,`ANTHROPIC_BASE_URL`  |

    To use Anthropic with Vizro-AI, you must have an account with paid-for credits available. None of the free accounts will suffice. You can check [all available Anthropic models including pricing on their website](https://docs.anthropic.com/en/docs/about-claude/models). Before usage, please review Anthropic guidelines on risk mitigation to understand potential model limitations and best practices.

>     More details on responsible usage can be found on this [link](https://trust.anthropic.com/).

Reviewer 2: WHat about this link: https://support.anthropic.com/en/collections/4078535-trust-safety

    | Base API URL      | `MISTRAL_BASE_URL`                        |

    To use Mistral with Vizro-AI, you can either use their API, which comes with [an associated cost](https://mistral.ai/technology/#pricing), or you could use their models for free under the Apache 2.0 license. In that case you need to setup the model API yourself. You can check [all available Mistral models including pricing on their website](https://docs.mistral.ai/getting-started/models/models_overview). This will also explain which version the below string acronyms currently point to.

>     Before usage, please review Mistral guidelines on risk mitigation to understand potential model limitations and best practices. More details on responsible usage can be found on this [link](https://docs.mistral.ai/capabilities/guardrailing/#guardrailing-results-with-mistral-safety-prompt).

Reviewer: ```suggestion
    Before using a model, please review Mistral guidelines on risk mitigation to understand potential model limitations and best practices. [See the Mistral site for more details on responsible usage](https://docs.mistral.ai/capabilities/guardrailing/#guardrailing-results-with-mistral-safety-prompt).
```

Reviewer 2: The problem with this link is that I am not sure we have that enabled atm? The other two vendors have more general advice, but here it is really about the API call. Should we enable this? And is there potentially more general info?

    | Base API URL      | `MISTRAL_BASE_URL`                        |

    To use Mistral with Vizro-AI, you can either use their API, which comes with [an associated cost](https://mistral.ai/technology/#pricing), or you could use their models for free under the Apache 2.0 license. In that case you need to setup the model API yourself. You can check [all available Mistral models including pricing on their website](https://docs.mistral.ai/getting-started/models/models_overview). This will also explain which version the below string acronyms currently point to.
    Before usage, please review Mistral guidelines on risk mitigation to understand potential model limitations and best practices.

>     [See the Mistral site for more details on responsible usage](https://help.mistral.ai/en/collections/280418-legal-security-and-data-privacy/).

Reviewer 2: Not sure this link applies, do you think this one might be better? https://help.mistral.ai/en/collections/272960-le-chat
